internal/pkg/monitor/monitor.go

```diff
@@ -186,12 +186,12 @@ func (m *simpleMonitorT) loadCheckpoint() sqn.SeqNo {
 
 // Run runs monitor.
 func (m *simpleMonitorT) Run(ctx context.Context) (err error) {
-	m.log.Info().Msg("start")
+	m.log.Info().Msg("Starting index monitor")
```

Comment:
Typically add a key to m.log showing context.  Unfortunately elastic logging UI does a not great job of showing other keys besides messages, and won't necessarily index the fields.  This makes it difficult to use structured logging properly.

Reply:
That is what I struggled during debugging today. There are so many other meta fields, the relevant ones are hard to find. Also I think there is some value to have the most important info also available directly in the message itself (again, personal preference).

Reply:
I would prefer if elastic logging supported a better way to aggregate actual messages from the application.  All the information is there, it is annoying and wasteful to have to duplicate it because the UI doesn't handle structured logging well.

Zerolog does a nice job with its pretty print on fields specifically from the app.  Would like to see elastic do something like that.

That said, is what it is.

Reply:
@weltenwort Any ideas on how to "fix" the above in the Logs UI?

Reply:
Yes, actually. But it's a bigger effort, obviously :grimacing: My vision would be that an integration could supply some form of configuration that the Logs UI can use to optimize the UX for these structured logs.

It's beyond of the scope of a comment in a PR review, but with the reconstitution of the infra ui team we might be able to get around to following up on that :crossed_fingers:

cc @mukeshelastic

Reply:
@weltenwort Great to hear, please pull me in if this conversation continues somewhere, happy to chime in.